docs/messaging/rabbitmq.md

```diff
@@ -12,8 +12,15 @@ The Semantic Conventions for [RabbitMQ](https://www.rabbitmq.com/) extend and ov
 
 ## RabbitMQ attributes
 
+### Destination name
+
 In RabbitMQ, the destination is defined by an *exchange* and a *routing key*.
-`messaging.destination.name` MUST be set to the name of the exchange. This will be an empty string if the default exchange is used.
+`messaging.destination.name` MUST be set according to the following rules:
+
+- Both *exchange* and *routing key* present and non-empty: `{exchange}:{routing key}`
```

Comment:
I think we have a problem on the consumer (even before this PR). 

![image](https://github.com/user-attachments/assets/371c4c46-e63c-4fa5-8c7a-3bf6d38b165b)

If someone publishes a message to exchange with a routing key and then there are bindings that route messages with this key to a queue (or queues) with a different name, then we have 3 things on the consumer side:

- we still have access to exchange and routing key - https://www.rabbitmq.com/docs/consumers#message-properties
- plus we have a queue name from which we got the message

and we have just two attributes (`mesasging.destination.name` and `messaging.rabbitmq.routing_key`) to store them

Reply:
Yes, I also stumbled on the same thing, where does the queue fits into these two attributes.  To make matters worse, for example in the Java instrumentation the [Queue name is what is used for the Span name](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/rabbitmq-2.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rabbitmq/ReceiveRequest.java#L28), not the exchange nor the routing key.

Reply:
I pushed a change, let me know what you think. [b02ff2a](https://github.com/open-telemetry/semantic-conventions/pull/1531/commits/b02ff2ae3bb8d7054e82ab7504167a2d35801204)